<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FastMarchingView</class>
 <widget class="QWidget" name="FastMarchingView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QmitkDataStorageComboBox" name="DataSelector"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Sigma</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ctkSliderWidget" name="SigmaValueSlider" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Alpha</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ctkSliderWidget" name="AlphaValueSlider" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Beta</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ctkSliderWidget" name="BetaValueSlider" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Stopping Value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ctkSliderWidget" name="StoppingValueSlider" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Threshold</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ctkRangeWidget" name="ThresholdValueSlider" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QmitkPointListWidget" name="SeedWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QCheckBox" name="PreviewCB">
        <property name="text">
         <string>2D Preview</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header>QmitkDataStorageComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkPointListWidget</class>
   <extends>QWidget</extends>
   <header>QmitkPointListWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkRangeWidget</class>
   <extends>QWidget</extends>
   <header>ctkRangeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
